react-custom-scrollbars/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Type definitions for react-css-transition-replace 2.0.1
+// Project: https://github.com/malte-wessel/react-custom-scrollbars
+// Definitions by: David-LeBlanc-git <https://github.com/David-LeBlanc-git>
```

Comment:
Because he is original author, he posted it as comment in issues of react-custom-scrollbar